library/init/datatypes.lean

```diff
@@ -394,23 +394,23 @@ definition default_has_sizeof_eq (A : Type) (a : A) : @sizeof A (default_has_siz
 rfl
 
 attribute [instance]
-definition nat_has_sizeof : has_sizeof nat :=
+definition nat.has_sizeof : has_sizeof nat :=
```

Comment:
Right now I am generating `<ind_name>.has_sizeof` for every inductive type, and I am relying on finding these declarations in the C++ by appending `has_sizeof` to the name of the inductive type in question. I think we should handle this the same way we handle `.cases_on`, and protect to avoid the overloads. I will fix it tomorrow morning.